cmake/Macros.cmake

```diff
@@ -182,6 +182,9 @@ macro(sfml_add_example target)
     # set the target's folder (for IDEs that support it, e.g. Visual Studio)
     set_target_properties(${target} PROPERTIES FOLDER "Examples")
 
+    # set the Visual Studio startup path for debugging
+    set_target_properties(${target} PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
`${CMAKE_CURRENT_SOURCE_DIR}`, are you sure?

Reply:
Considering it's called from the examples' source dir every time, yes. Don't think there should or needs to be a new parameter to pass the path. Or which variable had you in mind?